packages/telnyx_webrtc/lib/model/call_state.dart

```diff
@@ -18,6 +18,8 @@ enum CallState {
   held,
 
   /// [reconnecting] The call is reconnecting - for this state a [NetworkReason] is provided.
+  /// A call will remain in this state for a maximum of 45 seconds before automatically
```

Comment:
Is 45 seconds the default value? We are setting this right?

Maybe let's change it to say something like 'A call will remain in this state for the time specified within the configuration used to log in. The default value is 60 seconds'